--- Makefile    2024-07-01 15:33:51.000000000 +0000
+++ Makefile_v2 2024-07-08 17:06:55.148160213 +0000
@@ -332,7 +332,7 @@
            pip \
            pipenv=="$(PIPENV_VERSION)" \
            virtualenv=="$(VIRTUALENV_VERSION)" \
-           wheel
+           wheel --break-system-packages
        if ! type rustup >/dev/null 2>&1; then \
                curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh; \
                source $$HOME/.cargo/env; \